octopusdeploy/resource_nuget_feed.go

```diff
@@ -0,0 +1,176 @@
+package octopusdeploy
+
+import (
+	"fmt"
+
+	"github.com/OctopusDeploy/go-octopusdeploy/octopusdeploy"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceNugetFeed() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNugetFeedCreate,
+		Read:   resourceNugetFeedRead,
+		Update: resourceNugetFeedUpdate,
+		Delete: resourceNugetFeedDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"feed_uri": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"enhanced_mode": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
I'm not sure what effect it has but in other parts we've made this `Required: true` and it looks like it work right if left out (it pulls in the default). I wonder if setting `Optional: true` allows the user to specify `null` as a value.

Reply:
I tried it out and I was wrong